### PR TITLE
🇩🇬 Rename gateways group

### PR DESCRIPTION
**Pull request by WRadoslaw**
- #3583

### CODE REVIEW

**Review by thesan — CHANGES_REQUESTED** [review]
Please also edit the `defaultGroups` in [@/app/pages/WorkingGroups/WorkingGroups](https://github.com/Joystream/pioneer/blob/91756b75233c955bab010f67a4705c5ecf5bb587/packages/ui/src/app/pages/WorkingGroups/WorkingGroups.tsx#L22). It's used to show WG info before the QN response. Currently it shows:
Screenshot from 2022-10-27 17-55-53

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/app/pages/WorkingGroups/WorkingGroups.tsx [hunk 1]
```diff
@@ -11,10 +12,26 @@ import { WorkingGroupsTabs } from './components/WorkingGroupsTabs'
 export const WorkingGroups = () => {
   const { groups } = useWorkingGroups()
 
+  const sortedGroups = useMemo(
+    () =>
+      !groups
```
Comment (thesan) [inline packages/ui/src/app/pages/WorkingGroups/WorkingGroups.tsx:17]: My bad it should still be:
```suggestion
      !groups.length
```

**Review by thesan — CHANGES_REQUESTED** [review]

File: packages/ui/src/app/pages/WorkingGroups/WorkingGroups.tsx (hunk 1 above)
Comment (WRadoslaw) [reply packages/ui/src/app/pages/WorkingGroups/WorkingGroups.tsx:17]: I've remembered to change it in deps array, but forgot to change it here :D

**Review by thesan — APPROVED** [review]
:100:

**Pull request merged**